internal/mqini/mqini_test.go

```diff
@@ -16,7 +16,9 @@ limitations under the License.
 package mqini
```

Comment:
Copyright of this file hasn't been changed.  Need to add ", 2019".

Reply:
Thanks. updated it now.

Reply:
Copyright statements need to be formatted with two dates: the first is the year the code was first available, and the second is the year of the most recent change.  If both years are the same, then you can just have one date.  So this statement needs to be as follows:

```
© Copyright IBM Corporation 2018, 2019
```

Reply:
Thanks, I have updated the copy right now.